.github/workflows/ci-dgraph-code-coverage.yml

```diff
@@ -0,0 +1,74 @@
+name: ci-dgraph-code-coverage
+on:
+  push:
+    branches:
+      - main
```

Comment:
you probably can enable `coverage`, it's basically ci-dgraph-tests with some additions.